client/src/styles/LeftAndMain_Subsites.css

```diff
@@ -0,0 +1,82 @@
+/**
+* Styling for the subsite actions section in the CMS
+*/
+#SubsiteActions {
+  position: absolute;
+  padding: 0;
+  margin: 0;
+  right: 0;
+  top: 0;
+  width: 350px;
+  text-align: right;
+  margin-right: 130px;
```

Comment:
margin: 0 130px 0 0;

Reply:
No, as this would override any other margin applied to those other dimensions. This specifically sets _only_ the right margin.